tests/CatalogTests/Helpers/DB2CatalogCursorTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Data.SqlTypes;
+using NuGet.Services.Metadata.Catalog.Helpers;
+using Xunit;
+
+namespace CatalogTests.Helpers
+{
+    public class DB2CatalogCursorTests
+    {
+        public class TheByCreatedMethod
```

Comment:
nit: these tests are identical between the two methods, you could de-dupe them by using inheritance or `Theory`

Reply:
Had to make the `Db2CatalogCursor` a public type again for this, but fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/22da5bfc272abc369d36ac8a05eeb1e7f210c607